packages/suite/src/components/suite/layouts/SuiteLayout/SubpageNavigation.tsx

```diff
@@ -13,7 +13,6 @@ import { AppNavigationTooltip } from '../../AppNavigation/AppNavigationTooltip';
 const Container = styled.div`
     position: sticky;
     top: ${HEADER_HEIGHT};
-    background: ${({ theme }) => theme.backgroundSurfaceElevation0};
```

Comment:
Because `SubPageNavigation` is under the `FiatHeader` in `WalletLayout`, it has higher default stacking order. And the opaque background overlays the blur, cutting it off.  
That would be a problem, but in this case the bgcolor seems useless. I didn't notice any visual difference, see screenshots.